### PR TITLE
Fix preprocessing normalization bug

### PR DESCRIPTION
**Pull request by rorlich**

### CODE REVIEW

**Comment by prasanthpul** [comment]
@rorlich can you take care of the CLA and we'll merge this

**Comment by vinitra** [comment]
@rorlich -- ping for CLA.

File: onnx-ecosystem/inference_demos/resnet50_modelzoo_onnxruntime_inference.ipynb [hunk 1]
```diff
@@ -369,4 +371,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
+}
```
Comment (prasanthpul) [inline onnx-ecosystem/inference_demos/resnet50_modelzoo_onnxruntime_inference.ipynb]: extra chacacter?

**Comment by rorlich** [comment]
Sorry, Totally forgot to fix

File: onnx-ecosystem/inference_demos/resnet50_modelzoo_onnxruntime_inference.ipynb (hunk 1 above)
Comment (rorlich) [reply onnx-ecosystem/inference_demos/resnet50_modelzoo_onnxruntime_inference.ipynb]: probably.. should I revert that?

**Comment by rorlich** [comment]
I've signed the cla but i see this msg
"You have signed the CLA already but the status is still pending? Let us recheck it."

**Comment by prasanthpul** [comment]
It's not registering because the commits are done with the "Sagi Rorlich" account but the CLA is signed with "rorlich".

**Comment by rorlich** [comment]
i've merged that user ("Sagi Rorlich") with my active account.. hope it is now mergeable

File: onnx-ecosystem/inference_demos/resnet50_modelzoo_onnxruntime_inference.ipynb (hunk 1 above)
Comment (prasanthpul) [reply onnx-ecosystem/inference_demos/resnet50_modelzoo_onnxruntime_inference.ipynb]: yes, please remove the extra character

**Comment by prasanthpul** [comment]
@rorlich yes, the CLA looks good now.  can you update the extraneous character that was introduced? then it's good to merge

**Comment by rorlich** [comment]
done

**Pull request merged**